recipe/meta.yaml

```diff
@@ -37,7 +38,6 @@ requirements:
     - nlohmann_json
     - cppzmq
     - zlib
-    - zstd =1.5.2
```

Comment:
It doesn't work if `zstd-static` is in `host`?

Reply:
We did not try it yet. I will try it. It looks like currently platforms apart from Linux pass.